### ISSUE TITLE
[help required] Auto Update Widget

### ISSUE THREAD

**Issue by vishallanke** [reportportal/reportportal#1351]
Our project is a speedboat project where Pipelines of multiple projects gets executed in QA repo and then we decide health of Product. Report Portal Dashboard is widely used by team from management. We found report portal very useful. Component Health, Cumulative Trend Chart are very useful. However, these charts are not Auto Updated. It might be because updates takes time, so report portal team must have disabled auto update.

But, its creating too much confusion here. Lets say, Overall status shows 97% passed and cumulative trend showing something else because its not updated. Someone needs to perform updates manually. As stakeholders are from management team, there are chances that they could not see the last updated flag.

Is there a way/ hack to enable Auto Update ?

**Comment by MaryiaRymarchyk** [comment]
hI @vishallanke , sorry for the silence.
Actually we have two types of widget, let’s say common widgets and widgets with materialized view. Most of our widgets are common and they have automatic refresh each 60 seconds, also you can refresh each widget manually or reload browser page to have them updated.
There are only 2 widgets with materialized view: ‘Cumulative trend chart’ and ‘Component health check (table view)’. These widgets use temp tables and have huge background calculations, to avoid performance issues - we don’t refresh them automatically. Such kind of widgets can be updated only manually by clicking ‘Update’ button in the widget header, even browser refresh and re-login won’t update these widgets.
To sum up, almost all widgets are updated automatically each minute, except 2: ‘Cumulative trend chart’ and ‘Component health check (table view)’. And due to performance these two can't be updated automatically.
Let me know if you have more questions here or the issue can be closed.
